### PR TITLE
fast-als typing

### PR DESCRIPTION
**Pull request by elias-fauser**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/fast-als/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,116 @@
+// Type definitions for fast-als 1.0
+// Project: https://github.com/thorough-developer/fast-als
+// Definitions by: Elias Fauser <https://github.com/elias-fauser>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+/// <reference types="node" />
```
Comment (peterblazejewicz) [inline types/fast-als/index.d.ts:5]:
```suggestion
```
why this one is needed? (we're trying to avoid overload dependencies between DT packages)

Comment (elias-fauser) [reply types/fast-als/index.d.ts]: Sry this was a copy and paste error. Removed it

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@elias-fauser thanks!

**Comment by elias-fauser** [comment]
Ready to merge

**Pull request merged**